### ISSUE TITLE
Running a single test runs all tests

### ISSUE THREAD

**Issue by deadeyejoe** [BetterThanTomorrow/calva#1981]
Hi, suspect this is due to my workspace being structured as a polylith. EDIT: turns out this affects all projects.

When I add my test namespaces to the class path, running one test (the one under cursor) causes all tests in all test namespaces on the path to run. 

If I don't add them to the classpath, and try to run a single test, then all tests in the current namespace run.

**Comment by PEZ** [comment]
Hi! Thanks for reporting! I made some change to the test runner the other day that might have caused this. I don't really see how it caused it, but this used to work correctly, also in Polylith projects.

You are welcome to have a look at why this is happening, I'm pretty sure it's an easy fix once we have figured the cause out. Don't worry if you don't have the time. I'll have a look at it, if so. Anyway, if you *do* have the time and energy: There's a page on the wiki for how to [Hack on Calva](https://github.com/BetterThanTomorrow/calva/wiki/How-to-Hack-on-Calva) (and it is pretty straight forward to get started).
